eng/common/scripts/Create-APIReview.ps1

```diff
@@ -68,67 +75,82 @@ else
     exit(1)
 }
 
-$responses = @{}
-if ($packages)
-{
-    foreach($pkg in $packages.Keys)
-    {
-        Write-Host "Submitting API Review for package $($pkg)"
-        Write-Host $packages[$pkg]
-        $responses[$pkg] = Submit-APIReview -packagename $pkg -filePath $packages[$pkg] -uri $APIViewUri -apiKey $APIKey -apiLabel $APILabel
-    }
-}
-else
-{
-    Write-Host "No package is found in artifact path to submit review request"
-}
-
-$FoundFailure = $False
+# Check if package config file is present. This file has package version, SDK type etc info.
 if (-not $ConfigFileDir)
 {
     $ConfigFileDir = Join-Path -Path $ArtifactPath "PackageInfo"
 }
-foreach ($pkgName in $responses.Keys)
-{    
-    $respCode = $responses[$pkgName]
-    if ($respCode -ne '200')
+
+if ($packages)
+{
+    foreach($pkgPath in $packages.Values)
     {
+        $pkg = Split-Path -Leaf $pkgPath
         $pkgPropPath = Join-Path -Path $ConfigFileDir "$PackageName.json"
         if (-Not (Test-Path $pkgPropPath))
         {
             Write-Host " Package property file path $($pkgPropPath) is invalid."
+            continue
         }
-        else
+        # Get package info from json file created before updating version to daily dev
+        $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
+        $version = [AzureEngSemanticVersion]::ParseVersionString($pkgInfo.Version)
+        if ($version -eq $null)
         {
-            $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
-            $version = [AzureEngSemanticVersion]::ParseVersionString($pkgInfo.Version)
-            Write-Host "Package name: $($PackageName)"
-            Write-Host "Version: $($version)"
-            Write-Host "SDK Type: $($pkgInfo.SdkType)"
-            if ($version.IsPrerelease)
+            Write-Host "Version info is not available for package. Please check if version follows Azure SDK package versioning guielines."
```

Comment:
```suggestion
            Write-Host "Version info is not available for package $PackageName, because version '$(pkgInfo.Version)' is invalid. Please check if the version follows Azure SDK package versioning guidelines."
```